src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -1231,13 +1231,13 @@ public void fetchParticipationsWithSubmissionsAndResultsForCourses(List<Course>
     }
 
     /**
-     * Post /courses/:courseId/editors/:editorLogin : Add the given user to the editors of the course so that the student can access the course administration
+     * POST courses/:courseId/editors/:editorLogin : Add the given user to the editors of the course so that the student can access the course administration
```

Comment:
```suggestion
     * Post /courses/:courseId/editors/:editorLogin : Add the given user to the editors of the course so that the student can access the course administration
```